src/main/java/mate/academy/dao/BookDao.java

```diff
@@ -0,0 +1,18 @@
+package mate.academy.dao;
+
+import java.util.List;
+import java.util.Optional;
+import mate.academy.entity.Book;
+
+public interface BookDao {
+    Book create(Book book);
+
+    Optional<Book> findById(int id);
```

Comment:
The `findById` method should use `Long` for the `id` parameter instead of `int`, as specified in the task requirements.